API_request_test.py

```diff
@@ -87,4 +87,49 @@
 print(hours_2)
 
 print(temp_c)
-print(temp_c_2)
\ No newline at end of file
+print(temp_c_2)
+
+
+#####################
+### meteostat.net ###
+#####################
+
+# You need to register on the web page first to get API key
+# https://rapidapi.com/meteostat/api/meteostat/pricing
+# The url is set to get forecast data. For historical data it should be changed
+
+url_3 = 'https://meteostat.p.rapidapi.com/point/hourly'
+API_KEY_3 = <paste here your personal API key>
+
+params_3 = {
+	"lat": lat,
+	"lon": lon,
+	"start":"2024-02-28", #here we have to use some timestump to do it automaticaly
+    "end":"2024-02-28"
+}
+
+headers_3 = {
+	"X-RapidAPI-Key": <paste here your personal API key>,
+	"X-RapidAPI-Host": "meteostat.p.rapidapi.com"
+}
+
+res_3 = requests.get(url_3, headers= headers_3, params= params_3)
+res_3_dict = res_3.json()
+
+hours_3 = []
+temp_c_3 = []
+rain_3 = []
```

Comment:
Is there any particular reason to use separate lists for that? If I may suggest something, I'd change that to one nested dictionary with the following structure:
```
{
    <hour>: {
        temp: <value>,
        rain: <value>
    },
    ...
}
```
...to have all the gathered data grouped by hours. But I'll leave it up to you.

Reply:
I used separate lists for each weather component for a few reasons.  First, it is the data format I get from the other two API services. In both cases data are combined by weather component and not by hour. Secondly, the list structure is clearer for me in this case than the dictionary (maybe it is a lack of practice on my side). I also think that it will be easier for me to work with data in this format when I will be constructing a data frame and play with it using pandas. That is why I would use it this way.

Reply:
Understood, thanks for the explanation. You are right that some tools also may require/be easier to use with that data. Anyways, this will be confirmed when we jump into the data formatter/parser component, and it should not be the requester's responsibility.